### ISSUE TITLE
Keyboard navigation can lead to incorrect location

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/molecules-and-light#211]
**Test Device**

Bates

**Operating System**

Windows 7

**Browser**

Internet Explorer

**Problem Description**

Not for any test, but discovered during https://github.com/phetsims/QA/issues/419.

There is a scenario in which the keyboard navigation focus won't be on the active flashlight radio button upon keyboard navigating to the radio button group. (Please let me know if this doesn't make sense and I can record a video or reword this.)

**Steps to Reproduce**

1. Keyboard navigate to the radio buttons for the flashlight.
2. Change the flashlight to visible.
3. Shift+tab back to the actual flashlight.
4. Tab forward to the radio buttons for the flashlight. (It doesn't tab to visible, it tabs to microwave.)

**Comment by jessegreenberg** [comment]
I am not able to produce this, at least on Windows 10 IE11. Just pasting a gif of the above steps in IE11, with focus on the selected radio button. Do I have the steps correct @lmulhall-phet?
![4446c398bb87c6fe46a4373aea08dbb6](https://user-images.githubusercontent.com/6396244/64751332-c50ddf80-d4e9-11e9-94d8-053094c81fc6.gif)

**Comment by jessegreenberg** [comment]
OK thanks - I suspect that this could be a native behavior of Windows 7. @loganbraywork or @lmulhall-phet can you see if this happens in this test case?

https://jsfiddle.net/8ke1cb76/1/

Use keyboard nav to select the second radio button then tab/shift tab back and forth between the "Focus Me!" buttons. Do you see this behavior there?

**Comment by KatieWoe** [comment]
Still happens in the above QA issue.
The link provided in the above comment doesn't work in IE.

**Comment by jessegreenberg** [comment]
Ah, I see that JSfiddle doesn't work in IE anymore. Ill make another test.

**Comment by jessegreenberg** [comment]
@KatieWoe or @lmulhall-phet can you please see if it happens in Windows 7 here?

https://phet-dev.colorado.edu/html/jg-tests/test.html

**Comment by KatieWoe** [comment]
First link works ok.  Published link has the same bug but it seems worse, after switching once, the light selector loses focus, making it hard to make a selection.

**Comment by jessegreenberg** [comment]
I see, thanks - I made an issue about the issue with the published version. (https://github.com/phetsims/molecules-and-light/issues/222)

> First link works ok.

Good deal, does it have the behavior described in https://github.com/phetsims/molecules-and-light/issues/211#issue-492504711?

**Comment by KatieWoe** [comment]
No, it works as you would expect (sorry, that's what I meant)

**Comment by jessegreenberg** [comment]
OK, thank you. Win 7 bugs will be expensive for us to debug so we should decide whether or not to do this. @lmulhall-phet or @KatieWoe  do you think you could paste a gif here of the issue so we can discuss with the team?

**Comment by KatieWoe** [comment]
From original QA issue:
![newissue](https://user-images.githubusercontent.com/41024075/66862512-4542bd00-ef4e-11e9-8875-6ce1ef80f389.gif)

From published sim:
![published](https://user-images.githubusercontent.com/41024075/66862519-48d64400-ef4e-11e9-93f0-6e903d8090c4.gif)

**Comment by jessegreenberg** [comment]
Great, thank you @KatieWoe!

**Comment by jessegreenberg** [comment]
I remember that we may be able to set up remote debugging, @KatieWoe will help me with this to learn more.

**Comment by jessegreenberg** [comment]
@KatieWoe helped me set up a remote debugging session, writing notes from that here. I was able to verify the issue in the sim and see that it is not a problem in basic HTML like this:
<details><summary>Example HTML</summary>

```html
<!DOCTYPE html>


<html>

 
  <head>
     <meta charset="UTF-8">

  
    <title>title</title>

      
  </head>
   <body>

	<button>First Target</button>

	<label for="1">First Button</label>
	<input type="radio" name="set-1" id="1">

	<label for="2">Second Button</label>
	<input type="radio" name="set-1" id="2">

	<label for="3">Third Button</label>
	<input type="radio" name="set-1" id="3">

	<label for="4">Fourth Button</label>
	<input type="radio" name="set-1" id="4">
   
	<button>Second Target</button>
         
  </body>
 </html>

```
</details>

**Comment by jessegreenberg** [comment]
THis is a low priority issue, it only appears on Windows 7. I won't get to this for a while. I may investigate again closer to sim publication.

**Comment by terracoda** [comment]
Unsupported platform. Won't fix.
